### ISSUE TITLE
Using v1.0.0 fails to get correct SDK requested

### ISSUE THREAD

**Issue by timheuer** [actions/setup-dotnet#61]
This:
```
      - name: Setup .NET Core SDK
        uses: actions/setup-dotnet@v1.0.0
        with:
          dotnet-version: 3.1.100
```
fails to actually get the 3.1.100 SDK, however:
```
      - name: Setup .NET Core SDK
        uses: actions/setup-dotnet@v1
        with:
          dotnet-version: 3.1.100
```
works correctly.

The @v1.0.0 is what is suggested use on the marketplace listing.

**Comment by ZEisinger** [comment]
Thanks Tim, we are working to publish a new version to the marketplace with the latest updates.

**Comment by ZEisinger** [comment]
v1.4.0 has been published to the marketplace. Thanks for bringing up this problem.

**Comment by xt0rted** [comment]
@ZEisinger version 1.4.0 was published as `1.4.0` while prior versions were `v1.x.x` (both the marketplace & tags). If you try changing your pinned action from `v1.2.0` to `v1.4.0` your workflow crashes because it can't find the requested version.

```
Download action repository 'actions/checkout@v1.2.0'
Download action repository 'actions/setup-node@v1.2.0'
Download action repository 'actions/setup-dotnet@v1.4.0'
##[warning]Failed to download action 'https://api.github.com/repos/actions/setup-dotnet/zipball/v1.4.0'. Error Response status code does not indicate success: 404 (Not Found).
##[warning]Back off 14.584999999999999 seconds before retry.
##[warning]Failed to download action 'https://api.github.com/repos/actions/setup-dotnet/zipball/v1.4.0'. Error Response status code does not indicate success: 404 (Not Found).
##[warning]Back off 24.421 seconds before retry.
##[error]Response status code does not indicate success: 404 (Not Found).
```

**Comment by ZEisinger** [comment]
@xt0rted Thank you! This has been updated now.
